frontend_tests/node_tests/filter.js

```diff
@@ -184,6 +184,11 @@ run_test("basics", () => {
     assert(filter.contains_only_private_messages());
     assert(!filter.has_operator("search"));
     assert(filter.can_apply_locally());
+
+    assert(Filter.check_operand_valid("stream", "foo"));
+    assert(!Filter.check_operand_valid("streams", "foo"));
+    assert(Filter.check_operand_valid("in", "home"));
+    assert(!Filter.check_operand_valid("in", "foo"));
```

Comment:
I think we probably want to add `assert(Filter.check_operand_valid(operators)` for each of the sets of operators in the test blocks in this file; that will help ensure that if we add any new operators, which will naturally have tests, that when we copy/paste the test block, we verify that we update this function too.

(And it's also just a really valuable sanity check that you implemented the function correctly).

Reply:
Sure.